infrastructure/modules/dns.bicep

```diff
@@ -0,0 +1,21 @@
+param environment string
+
+param webappAzureGeneratedFqdn string
+
+resource zone 'Microsoft.Network/dnsZones@2018-05-01' = {
```

Comment:
This DNS zone already exists from the shared azure repository - we should include the `existing` keyword to look up the existing resource